.github/workflows/pypi-publish.yml

```diff
@@ -29,6 +29,30 @@ jobs:
         python-minor-version: ${{ matrix.python-minor-version }}
         token: ${{ secrets.PYPI_TOKEN }}
         repo: "pypi"
+  linux-arm:
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        python-minor-version: [ "11" ]
```

Comment:
Why so recent? Should this be set to minimal supported version?

Reply:
![image](https://github.com/lancedb/lance/assets/43097991/0a76c52f-667f-4dd5-ac58-888014fce7d7)

Reply:
Oh wait, I might've accidentally made that change. I asked Chang about `python.yml`, not this one

Reply:
Oh, I might've accidentally made that change. I asked Chang about the `python.yml` file, so should this one include all versions?

Reply:
IIUC, if we build with 3.11, it will be compatible with Python >= 3.11. If we build with Python 3.8, it will be compatible with Python >= 3.8. Though I think you can check by verifying what platform tag the resulting wheel has. We'll want `cp38` and not `cp311`.